cli/flox/src/commands/containerize/macos_containerize_proxy.rs

```diff
@@ -1,21 +1,115 @@
 use std::convert::Infallible;
 use std::path::PathBuf;
 
+use dirs::home_dir;
+use flox_rust_sdk::flox::FLOX_VERSION;
 use flox_rust_sdk::providers::container_builder::{ContainerBuilder, ContainerSource};
 
-#[allow(unused)]
-struct ContainerizeProxy {
+use super::Runtime;
+use crate::config::FLOX_DISABLE_METRICS_VAR;
+
+const FLOX_FLAKE: &str = "github:flox/flox";
+const FLOX_PROXY_IMAGE: &str = "ghcr.io/flox/flox";
```

Comment:
How come `ghcr.io/flox/flox` instead of `nixos/nix`?

Reply:
It's described a bit further down:

>         // Use a released Flox container of the same semantic version as a base
>         // because it already has:
>         //
>         // - most of the dependency store paths
>         // - substitutors configured
>         // - correct version of nix
>         let flox_container = format!("{}:{}", FLOX_PROXY_IMAGE, flox_version_tag);
>         command.arg(flox_container);

It could be swapped for a `nixos/nix` container in or after https://github.com/flox/flox/issues/2469 if we're able to satisfy all of those.